ui/eureka/eureka.js

```diff
@@ -2139,6 +2611,40 @@ class EurekaTracker {
     }
   }
 
+  OnCE(e) {
+    // Upon entering Eureka we usually receive the CE info before
+    // this.zoneInfo is loaded, so lets store the events until we're
+    // able to process them.
```

Comment:
Is there some way to not make pop noises for CEs that have already started?

Reply:
Yeah that should be simple to implement by passing the ceStatus as an optional parameter to onFatePop.